client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -117,15 +121,8 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 	}
 }
 
-export default connect( ( state, { siteId, steps } ) => {
-	const tasks = compact( [] );
```

Comment:
🔪 Just wasn't used?  🤷‍♂️

Reply:
Yeah, I think it was meant to serve as as stub for what ended up being handled by `stepsTodo`.